apax/data/preprocessing.py

```diff
@@ -119,7 +119,7 @@ def get_shrink_wrapped_cell(positions):
         if cell[idx, idx] < 10e-1:
             cell[idx, idx] = 1.0
 
-    cell.flat[:: cell.shape[1] + 1] += 1
+    cell[np.diag_indices_from(cell)] += 1
```

Comment:
so the purpose of this line is to increase the cell by 1 Angstrom in all directions.
is this necessary?

Reply:
Yes. If the cell is too tight some atoms will be counted as being in a adjacent cell and the neighbour list will be wrong.